hash/fmtowns_cd.xml

```diff
@@ -10640,14 +10896,13 @@ User/save disks that can be created from the game itself are not included.
 		<info name="release" value="199202xx" />
 		<info name="usage" value="Requires 2 MB RAM"/>
 		<part name="flop1" interface="floppy_3_5">
-			<feature name="part_id" value="User Disk" />
```

Comment:
Any reason for dropping the `part_id` here rather than correcting it if it’s wrong?  It’s generally more useful to have descriptive names than not to have them.

Reply:
I try to use the actual disk names if they are known, but in the case of Koei games like this one, the floppy disks only have the game title printed on the label, and nothing else. I didn't know that at the time I originally added this entry, so I thought it would be better to remove the part_id than to keep an unrealistic one.